service/controller/app/resource/releasemigration/create.go

```diff
@@ -69,7 +82,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return nil
 	}
 
-	if strings.ToLower(cr.Status.Release.Status) != helmclient.StatusDeployed {
+	if deploy.Status.ReadyReplicas > 0 {
```

Comment:
this is the point we are using chart-operator deployment status.

Reply:
Nice and good catch on finding this.